experimenter/experimenter/nimbus_ui_new/templates/nimbus_experiments/subscribers_list.html

```diff
@@ -0,0 +1,27 @@
+<tr class="subscribers-section">
+  <th>Subscribers</th>
+  <td id="subscribers-list" colspan="2">
+    {% for subscriber in experiment.subscribers.all %}
+      <p>{{ subscriber.email }}</p>
+    {% empty %}
+      <span class="text-danger">Not set</span>
+    {% endfor %}
+  </td>
```

Comment:
Probably a concern that can be dealt with later if it becomes a problem, but perhaps this should get a max height and overflow auto or some other way to ensure the list of subscribers isn't huge?

Or, alternatively, is there a good reason to show every user the list of every subscriber? Could probably just show the button so they know whether they are subscribed or not.

Reply:
Good point, @mikewilli, for longer list support, I have added the CSS. For the second point, I am not sure design-wise. I am taking note of this and will ask Shell, too, so we can think more about this when we start redesigning.